node/pkg/logscribeconsumer/app.go

```diff
@@ -169,12 +205,18 @@ func extractDbEntry(entry map[string]interface{}) ([]any, error) {
 
 	timestamp := time.Unix(int64(timeVal.(float64)), 0)
 	message := messageVal.(string)
-	levelStr := levelStrVal.(string)
 
-	level, err := zerolog.ParseLevel(levelStr)
+	var levelStr string
+	if a.Level == "" {
+		levelStr = levelStrVal.(string)
+	} else {
+		levelStr = a.Level
+	}
```

Comment:
I think the logic should be updated, it should be filtering out logs which have lower level than `a.Level`
let me know if I'm missing anything

Reply:
please check and update code in 
`isLogLevelValid` func to utilize `a.Level` property

Reply:
you'll probably not need the code here